cfme/tests/physical_infrastructure/ui/test_physical_server_list_buttons.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+import pytest
+from selenium.common.exceptions import StaleElementReferenceException
+
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.common.physical_server_views import (
+    PhysicalServersView,
+)
+from cfme.utils.wait import wait_for
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="module")]
+
+
+@pytest.fixture(scope="module")
+def physical_servers_collection(appliance, setup_provider_modscope):
+    # Get and return the first physical server
+    return appliance.collections.physical_servers
+
+
+@pytest.fixture(scope="module")
+def physical_servers(appliance, provider):
```

Comment:
This fixture is unnecessary - again there's no setup happening here.  Additionally the provider fixture is a PhysicalServerProvider object, there's no need to try to re-instantiate it through the collection.